package.json

```diff
@@ -1,55 +1,65 @@
 {
-  "name": "dockercomposator-plugin",
+  "name": "docker-composator-plugin",
   "version": "0.1.0",
-  "description": "DockerCompose plugin for Leto-Modelizer.",
-  "main": "dist/dockercomposator-plugin.js",
-  "exports": "./dist/dockercomposator-plugin.js",
+  "description": "Plugin for creating docker compose files",
+  "main": "dist/docker-composator-plugin.js",
+  "exports": "./dist/docker-composator-plugin.js",
   "scripts": {
     "build": "webpack",
     "build:docs": "jsdoc --readme README.md --package package.json -r ./src/ -c jsdoc.config.json -d docs",
-    "lint": "eslint --max-warnings=0 --ext .js src",
-    "lint:fix": "eslint --max-warnings=0 --fix --ext .js src",
-    "lint:report": "eslint --max-warnings=0 --ext .js src -f json-relative > eslint.json",
+    "lint": "eslint --ext .js src",
+    "lint:fix": "eslint --fix --ext .js src",
+    "lint:report": "eslint --ext .js src -f json-relative > eslint.json",
     "prepare:docs": "sed -i 's#taffydb#@jsdoc/salty#g' node_modules/better-docs/publish.js",
     "test": "jest",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "generate:parser": "node scripts/generate_parser.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ditrit/dockercomposator-plugin.git"
   },
   "keywords": [
     "Leto-modelizer",
-    "dockercomposator-plugin",
-    "docker-compose"
+    "docker-compose",
+    "docker-composator-plugin"
+  ],
+  "authors": [
```

Comment:
In ditrit we use authors as maintener so keep me in this list. Remove other names